module/modmanager/manager.go

```diff
@@ -747,6 +748,7 @@ func (m *module) registerResources(mgr modmaninterface.ModuleManager, logger gol
 			}
 		case api.API.IsService():
 			for _, model := range models {
+				logger.Debugf("registering service from module", "module", m.name, "API", api.API, "model", model)
```

Comment:
```suggestion
				logger.Debugw("registering service from module", "module", m.name, "API", api.API, "model", model)
```

Reply:
Hmm I did `Debugw` for the structured logging. Do you think `Debugf` would be better here? My understanding was that we should bias toward structured logging where possible.